### ISSUE TITLE
Only show sender name in editor if more than one available

### ISSUE THREAD

**Issue by jowlo** [tutao/tutanota#3116]
Due to popular demand™ he editor now always shows the sender and a selector - even if there is only a single alias available.

If there is no alternative available, we should either hide the sender field altogether (my favourite) or at least remove the dropdown on the element and just list the sender name.

**Comment by charlag** [comment]
I am against removing the field because people will be confused. Deactivating is fine I think

**Comment by bedhub** [comment]
I am also in favor of keeping the selector. Maybe we can add an option to add / edit an email alias.

**Comment by jowlo** [comment]
Yes, that sounds like a good idea. I just found the selector with a single choice not very much selector-like. Adding an "Add" option that directs you to where to buy aliases sounds good. It could be a "+" sign but i think we should keep the "direct" UI of the editor as clean as possible.
